routes/routes.go

```diff
@@ -27,8 +26,17 @@ type Clients struct {
 
 // Setup registers routes for the service
 func Setup(ctx context.Context, r *mux.Router, cfg *config.Config, c Clients, cacheList cache.List) {
-	log.Info(ctx, "adding routes")
```

Comment:
Why is this being removed?

Reply:
Was removed during testing and wasn't put back in. Added back now